docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+# BuildKit Roadmap (tentative)
+
+This document roughly describes the roadmap of the BuildKit project.
+We will be using GitHub Projects and GitHub Milestones for more detailed roadmap.
+
+## Task 1 (2018Q1)
+- Implement all features needed for replacing the legacy builder backend of moby-engine
+
+- Test, test, and test (help needed!)
+
+- Integrate BuildKit to moby-engine as an experimental builder backend. (`moby-engine --experimental --build-driver buildkit`)
+-- Probably for Linux only
+
+## Task 2 (2018Q2-Q3)
+
+- Promote BuildKit to the default moby-engine build backend
```

Comment:
@tonistiigi 

I guess this task requires integrating the containerd image store to moby-engine first?

Reply:
Isn't it possible to export images to the current engine image storage using `docker import` ?

Reply:
@dnephin It is, but it has quite a significant performance penalty because data needs to be copied.

@AkihiroSuda I agree. Should it get very delayed we could consider implementing the snapshotter dependency with moby storage drivers but let's try the correct way first.

Reply:
@mlaventure Do you know when we will integrate the containerd image store & snapshot subsystem to Moby-engine?

Reply:
I do not, @stevvooe & @dmcgowan would have a better idea.

Reply:
The plan is still that early next year. The snapshotters will be available to use before then. Something to consider is that having integration with the containerd snapshotter does not necessarily mean the storage drivers will not continue to be used. I don't know what the plan will be but it might be wise to plan on using the snapshotters and having a way to export into an existing Docker storage driver.